source/jormungandr/jormungandr/interfaces/v1/serializer/time.py

```diff
@@ -48,13 +48,25 @@ def to_value(self, value):
             return ""
         return datetime.utcfromtimestamp(value).strftime('%H%M%S')
 
+    def _jsonschema_type_mapping(self):
+        return {
+            'type': 'string',
+            'pattern': 'd{2}\d{2}\d{2}'
+        }
+
 class DateTimeField(PbField):
     """
     custom date format from timestamp
     """
     def to_value(self, value):
         return timestamp_to_str(value)
 
+    def _jsonschema_type_mapping(self):
+        return {
+            'type': 'string',
+            'pattern': '\d{4}\d{2}\d{2}T\d{2}\d{2}\d{2}'
```

Comment:
I don't really know how to do this, but I would prefer to abstract a bit the JsonSchema stuff